io/transmat/gs/gs_token.go

```diff
@@ -0,0 +1,71 @@
+package gs
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/storage/v1"
+)
+
+/*
+ Environment variable used to directly pass an OAuth2 access token to repeatr
+*/
+const EnvGsAccessToken = "GS_ACCESS_TOKEN"
+
+/*
+ Environment variable used to pass the path of
+ a Google Developers service account JSON key file
+ which is then used for two-legged OAuth to create an Access Token
+*/
+const EnvGsAccountFile = "GS_SERVICE_ACCOUNT_FILE"
```

Comment:
We should probably add `"GOOGLE_APPLICATION_CREDENTIALS"` as an alias of this as well: https://github.com/golang/oauth2/blob/2baa8a1b9338cf13d9eeb27696d761155fa480be/google/default.go#L62  I like your names here better, TBH, and `GS_ACCESS_TOKEN` in particular is scoped to make sense, but if someone's expecting these defaults to work, might as well.

On the other hand, that `DefaultTokenSource` function also hits `metadata.OnGCE` and that in turn... That's... [rather indiscreet](https://github.com/GoogleCloudPlatform/gcloud-golang/blob/6fdcab499d2c45e2939521c4047855f12a745a1d/compute/metadata/metadata.go#L165) and not the sort of default behavior I want in this project -- phoning home without configuration, potentially adding hundreds of ms of latency in a timeout that can't be configured or disabled, for a feature that's for some people *never used*; all No Thanks.  So, maybe we should copy some of that functionality, and leave the rest out, or bring in a patched version that wraps that in conditionals.